### PR TITLE
Correcting steps for new admin center

### PR DESCRIPTION
**Pull request by get-itips**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/3828

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/set-up-communications-credits-for-your-organization.md [hunk 1]
```diff
@@ -53,9 +53,9 @@ For more information, see [Microsoft Teams add-on licensing](teams-add-on-licens
 
 1. Sign in to the new Office 365 portal with your work or school account.
     
-2. In the left navigation of the Microsoft 365 admin center, go to **Billing** > **Subscriptions** > **Add subscriptions**.
+2. In the left navigation of the Microsoft 365 admin center, go to **Billing** > **Purchase Services**, scroll down and select **Add-Ons**.
```
Comment (JohanFreelancer9) [inline Teams/set-up-communications-credits-for-your-organization.md]:
```suggestion
2. In the left navigation of the Microsoft 365 admin center, go to **Billing** > **Purchase Services**. Scroll down and select **Add-Ons**.
```

File: Teams/set-up-communications-credits-for-your-organization.md [hunk 2]
```diff
@@ -53,9 +53,9 @@ For more information, see [Microsoft Teams add-on licensing](teams-add-on-licens
...
 
-3. Expand **Add-on subscriptions**, and then choose **Communications Credits** > **Buy now**.
+3. Click to choose **Communications Credits**.
```
Comment (JohanFreelancer9) [inline Teams/set-up-communications-credits-for-your-organization.md]:
```suggestion
3. Select **Communications Credits**.
```

File: Teams/set-up-communications-credits-for-your-organization.md [hunk 3]
```diff
@@ -88,9 +88,9 @@ When you are setting up Communications Credits, you will need to investigate cal
     
 2. In the left navigation of the Microsoft 365 admin center, go to **Users** > **Active users**, and then select a user or users from the list.
     
-3. In the Action pane under **Product licenses**, click **Edit**.
+3. Click to choose **Licenses and Apps**.
```
Comment (JohanFreelancer9) [inline Teams/set-up-communications-credits-for-your-organization.md]:
```suggestion
3. Choose **Licenses and Apps**.
```

File: Teams/set-up-communications-credits-for-your-organization.md [hunk 4]
```diff
@@ -88,9 +88,9 @@ When you are setting up Communications Credits, you will need to investigate cal
...
     
-4. On the **Product licenses** page, toggle **Communications Credits** to **On** to assign this license, and then click **Save**.
+4. Toggle **Communications Credits** to **On** to assign this license, and then click **Save**.
```
Comment (JohanFreelancer9) [inline Teams/set-up-communications-credits-for-your-organization.md]:
```suggestion
4. Toggle **Communications Credits** to **On** to assign this license, and then select **Save**.
```

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**